components/molecule/phoneInput/demo/index.js

```diff
@@ -1,17 +1,9 @@
-import {useState} from 'react'
-
-import {
-  AntDesignIcon,
-  Article,
-  H1,
-  H2,
-  H3,
-  Paragraph
-} from '@s-ui/documentation-library'
+import {AntDesignIcon, H1, Paragraph} from '@s-ui/documentation-library'
 import AtomIcon from '@s-ui/react-atom-icon'
 
-import MoleculePhoneInput from '../src/index.js'
-import {phoneValidationType, PREFIXES} from '../src/settings.js'
+import ArticleDefault from './ArticleDefault.js'
+import ArticleStates from './ArticleStates.js'
+import ArticleTypes from './ArticleTypes.js'
```

Comment:
Great to split the demo in 3 different section/files. As a reminder, are we showing a lindline example? should we do it? It's not easy to imagine that reading the code 😓 . Thanks again!